examples/notebooks/FLARE.ipynb

```diff
@@ -0,0 +1,668 @@
+                        "it with the more satisfaction, I ceased my labours and sat down upon\n",
+                        "the bones. When at last the clanking subsided, I resumed the trowel,\n",
+                        "and finished without interruption the fifth, the sixth, and the seventh\n",
+                        "tier.  The wall was now nearly upon a level with my breast.  I again\n",
+                        "paused, and holding the flambeaux over the mason-work, threw a few\n",
+                        "feeble rays upon the figure within.\n",
+                        "\n",
+                        "A succession of loud and shrill screams, bursting suddenly from the\n",
+                        "throat of the chained form, seemed to thrust me violently back.  For a\n",
+                        "brief moment I hesitated--I trembled.  Unsheathing my rapier, I began\n",
+                        "to grope with it about the recess; but the thought of an instant\n",
+                        "reassured me.  I placed my hand upon the solid fabric of the catacombs,\n",
+                        "and felt satisfied.  I reapproached the wall; I replied to the yells of\n",
+                        "him who clamoured.  I re-echoed--I aided--I surpassed them in volume\n",
+                        "and in strength.  I did this, and the clamourer grew still.\n",
+                        ">>> USER INPUT: Who is Luchesi in relation to Antonio?\n",
+                        ">>> RESPONSE: \u001b[0m\n",
+                        "\n",
+                        "\u001b[1m> Finished chain.\u001b[0m\n",
+                        "\n",
+                        "\u001b[1m> Finished chain.\u001b[0m\n"
+                    ]
+                },
+                {
+                    "data": {
+                        "text/plain": [
+                            "' Luchesi is a rival connoisseur of wine to Fortunato. He is mentioned by Montresor as someone who could distinguish Amontillado from Sherry, but not as well as Fortunato.\\n\\n'"
+                        ]
+                    },
+                    "execution_count": 11,
+                    "metadata": {},
+                    "output_type": "execute_result"
+                }
+            ],
+            "source": [
+                "# Note that FLARE uses a larger prompt, so more complex questions that this may cause the token limit to be exceeded.\n",
+                "query = \"Who is Luchesi in relation to Antonio?\"\n",
+                "flare.run(query)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {
+                "nbmake": {
+                    "post_cell_execute": [
+                        "# Delete collection after test so other tests don't re-use or hit limit\n",
+                        "vstore.delete_collection()\n"
+                    ]
+                }
+            },
```

Comment:
Maybe don't make this a nbmake call, but actually expose it in the notebook so users can cleanup the created collection from their astra.

Reply:
I'll actually do both -- we want the CI to clean up, and I'll expose it for the users in a Cleanup section